src/main/scala/org/broadinstitute/dsde/agora/server/AgoraConfig.scala

```diff
@@ -8,6 +8,24 @@ import org.broadinstitute.dsde.agora.server.model.AgoraEntityType.EntityType
 
 object AgoraConfig {
   private val config: Config = ConfigFactory.load()
+  
+  val LocalEnvironment = "local"
+  val DevEnvironment = "dev"
+  val CiEnvironment = "ci"
+  val ProdEnvironment = "prod"
+
+  val Environments = List[String](LocalEnvironment, DevEnvironment, CiEnvironment, ProdEnvironment)
+
+  // TODO - what is the default??
```

Comment:
Can we remove this comment now? I think we decided that local would be the default, which is what you're doing.